rasa_nlu/server.py

```diff
@@ -31,12 +31,20 @@ def create_argument_parser():
 
     parser.add_argument('-e', '--emulate',
                         choices=['wit', 'luis', 'dialogflow'],
-                        help='which service to emulate (default: None i.e. use '
-                             'simple built in format)')
+                        help='which service to emulate (default: None i.e. use'
+                             ' simple built in format)')
     parser.add_argument('-P', '--port',
                         type=int,
                         default=5000,
                         help='port on which to run server')
+    parser.add_argument('-p', '--pre_load',
```

Comment:
let's not do a short argument (`-p`) and just use `--pre_load` (as `-p` might easily be confused with the port one)